openiddict/src/Audacia.Auth.OpenIddict.Seeding.EntityFramework/Audacia.Auth.OpenIddict.Seeding.EntityFramework.csproj

```diff
@@ -1,20 +1,21 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFramework>net6.0</TargetFramework>
+		<TargetFrameworks>$(CustomTargetFrameworks)</TargetFrameworks>
 		<OutputType>Exe</OutputType>
 		<Authors>Audacia</Authors>
 		<PackageIcon>package-icon.png</PackageIcon>
 		<Copyright>Copyright © Audacia $([System.DateTime]::UtcNow.Year)</Copyright>
 		<Description>.NET tool to seed OpenIddict applications and scopes in a database using Entity Framework 6.x as the ORM.</Description>
 		<PackageLicenseExpression>MIT</PackageLicenseExpression>
 		<PackageRequireLicenseAcceptance>true</PackageRequireLicenseAcceptance>
-		<Version>3.0.0</Version>
+		<Version>4.0.0</Version>
 		<Nullable>enable</Nullable>
 		<GenerateDocumentationFile>True</GenerateDocumentationFile>
 		<TreatWarningsAsErrors>True</TreatWarningsAsErrors>
 		<ToolCommandName>openiddict-ef6-seeding</ToolCommandName>
 		<PackAsTool>true</PackAsTool>
+		<GenerateProgramFile>false</GenerateProgramFile>
```

Comment:
Without this a couple of projects could not be built (at least in .NET 8) due to there being two Main methods